streams/src/test/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyKeyValueStoreTest.java

```diff
@@ -171,6 +208,10 @@ public void shouldThrowInvalidStoreExceptionDuringRebalance() throws Exception {
         rebalancing().get("anything");
     }
 
+    @Test(expected = InvalidStateStoreException.class)
+    public void shouldThrowInvalidStoreExceptionOnApproximateNumEntriesDuringRebalance() throws Exception {
+        rebalancing().approximateNumEntries();
```

Comment:
`rebalancing()` should never throw an `InvalidStateStoreException` as it is just constructing the `CompositeReadOnlyKeyValueStore` wrapper. The underlying stores should not be accessed until `get`, `range`, or `all` are called. 
So, i think this is safe to leave it as it is